tool/loadgen/main.go

```diff
@@ -53,32 +53,45 @@ func main() {
 	if cAdvisorURL != "" && containers == "" {
 		panic("flag -containers is required when -cadvisor is provided")
 	}
-	if strings.Contains(containers, ",") {
-		panic("not implemented: only one container name supported")
-	}
-	containerName := containers
 
 	log.Printf("Target is %q", targetURL)
 
 	waitUntilReady(targetURL, maxWait)
 	warmUp(targetURL, rps, warmupDuration)
 
-	stats := Stats{}
+	statsMap := make(map[string]Stats)
 	if cAdvisorURL != "" {
-		stats.Before = containerStats(cAdvisorURL, containerName)
+		for _, containerName := range strings.Split(containers, ",") {
+			imageName := strings.Split(containerName, "-")[0]
```

Comment:
Would be nice if we could query cAdvisor with a simpler, static, name. Then we could avoid some string manipulation.

I remember it wasn't obvious how to use the API 😓